src/api-client.ts

```diff
@@ -13,10 +13,7 @@ import RevAiApiJob from './models/RevAiApiJob';
 import RevAiApiTranscript from './models/RevAiApiTranscript';
 import RevAiJobOptions from './models/RevAiJobOptions';
 
-/* tslint:disable-next-line:no-var-requires */
-const versionNumber = require('../package.json').version;
```

Comment:
This import isn't possible with the current structure of the package. I can explain more in detail if anyone wants but currently it would require rearranging the package in a way which I don't think is intuituve